ivy/functional/frontends/numpy/manipulation_routines/splitting_arrays.py

```diff
@@ -6,3 +6,7 @@
 @to_ivy_arrays_and_back
 def dsplit(ary, indices_or_sections):
     return ivy.dsplit(ary, indices_or_sections)
+
+@to_ivy_arrays_and_back
+def vsplit(ary, indices_or_sections):
+    return ivy.vsplit(ary, indices_or_sections
```

Comment:
I think you have missed a bracket here 🙂 
```suggestion
    return ivy.vsplit(ary, indices_or_sections)
```